packages/fx-core/tests/core/middleware/projectMigrationV3.test.ts

```diff
@@ -622,6 +622,22 @@ describe("manifestsMigration", () => {
     await copyTestProject(Constants.manifestsMigrationHappyPath, projectPath);
     await fs.remove(path.join(projectPath, "templates/appPackage/aad.template.json"));
 
+    // Action
+    try {
+      await manifestsMigration(migrationContext);
+    } catch (error) {
+      assert.equal(error.name, "AadManifestTemplateNotExist");
+    }
+  });
+
+  it("happy path: projects created before 4.0.0", async () => {
```

Comment:
Projects created before 4.0.0 should using 2 different teams app manifest (remote/local), so this will also fail?